.pylintrc

```diff
@@ -278,7 +278,8 @@ module-naming-style=snake_case
 
 # Regular expression matching correct module names. Overrides module-naming-
 # style
-#module-rgx=
+# allowing exactly four digits at the beginning for migrations
+module-rgx=([0-9]{4})?([a-z_][a-z0-9_]*)$
```

Comment:
sorry that i'm bringing this up again: couldn't this just be put in the commit "Apply our changes to .pylintrc"? i mean there are no other changes related to this that would make a separate commit necessary.